src/Time.coffee

```diff
@@ -1,14 +1,18 @@
-
 class CountDown
+
   constructor: ({@time = 120, @delay = 1000}) ->
     @onComplete = ->
     @onTick = ->
+    @pauseTimer = false
   start: (immediate = false) ->
     {delay, onTick, onComplete} = @
     interval = null
     tick = ->
-      onTick and onTick()
-      @time -= 1
+      # NOTE [scollins] well, at least this if statement and flag condition
+      # doesn't break the timer lol
+      if not @pauseTimer
```

Comment:
this only sort of works, because the timer fires the tick at a set interval of time, when you "miss" ticks by using the condition, when the condition is true, it will not technically resume in the right timing.

The best way to handle pause/resume of a timer, is to clear the interval and recreate the interval and also fire the tick that way when the timer is told to resume, it will appear to start back up immediately, in time as the user expects.